File: day2/diagonal-difference/solution.py
def diagonalDifference(arr):
    primary = []
    secondary =[]
    size = len(arr)
    for i in range(0,size):
        primary.append(arr[i][i])
        secondary.append(arr[size-i-1][i])
    primary = sum(primary)
    secondary =sum(secondary)
    
    return abs(primary - secondary)

File: day2/diagonal-difference/solution.js
function diagonalDifference(arr) {
  let primary = 0;
  let secondary = 0;
  let size = arr.length;
  for (let i = 0; i < size; i++) {
    primary += arr[i][i];
    secondary += arr[size - i - 1][i];
  }

  return Math.abs(primary - secondary);
}
